layer demo/layer-demo.bas

```diff
@@ -10,17 +10,22 @@
 10 REM
 11 REM ==COPY LAYER 1 REGISTERS to LAYER 0==
 12 REM
-13 FOR I=0 TO 9 : A = VPEEK($F,$3000+I) : VPOKE $F,$2000+I,A : NEXT I
```

Comment:
Most of these changes were simple, such as changing VPEEK/VPOKE to PEEK/POKE using the VERA addresses now mapped directly.